### PR TITLE
Ci/ubuntu 20.04 GitHub action

### PR DESCRIPTION
**Pull request by hannes-steffenhagen-diffblue**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by danpoe — APPROVED** [review]

File: .github/workflows/pull-request-checks.yaml [hunk 1]
```diff
@@ -4,6 +4,28 @@ on:
...
+        run: sudo apt-get install -yq cmake ninja-build gcc g++ maven flex bison
+      - name: Configure using CMake
+        run: |
+          mkdir build
+          cd build
+          cmake .. -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++
+      - name: Build with Ninja
+        run: cd build; ninja
+      - name: Run tests
+        run: cd build; ctest . -V -L CORE
```
Comment (thomasspriggs) [inline .github/workflows/pull-request-checks.yaml:64]: Does this run the unit tests?

Comment (hannes-steffenhagen-diffblue) [reply .github/workflows/pull-request-checks.yaml:64]: yes

**Review by NlightNFotis — APPROVED** [review]
LGTM

**Review by chrisr-diffblue — APPROVED** [review]

File: .github/workflows/pull-request-checks.yaml [hunk 2]
```diff
@@ -4,6 +4,28 @@ on:
...
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Fetch dependencies
+        env:
+          # This is needed in addition to -yq to prevent apt-get from asking for
+          # user input
+          DEBIAN_FRONTEND: noninteractive
+        run: sudo apt-get install -yq cmake ninja-build gcc g++ maven flex bison
```
Comment (thomasspriggs) [inline .github/workflows/pull-request-checks.yaml]: :pick: Maybe add `libxml2-utils` for `xmllint` testing.

**Review by thomasspriggs — APPROVED** [review]

**Pull request merged**